CONTRIBUTING.rst

```diff
@@ -127,43 +124,6 @@ that your files start with a line like:
 
    # Copyright 20xx-present Kensho Technologies, LLC.
 
-Python 2 vs Python 3
---------------------
-
-In order to ensure that tests run with a fixed set of packages in both
-Python 2 and Python 3, we always run the tests in a virtualenv managed
-by pipenv. However, since some of our dependencies have different
-requirements for Python 2 and Python 3, we have to keep two pipenv
-lockfiles -- one per Python version.
-
-We have chosen to make the Python 3 lockfile the default (hence named
-:code:`Pipfile.lock`), since Python 3 offers better performance and we like
-our tests and linters running quickly. The Python 2 lockfile is named
-:code:`Pipfile.py2.lock`.
-
-If you need to set up a Python 2 virtualenv locally, simply run the
-following script:
-
-::
-
-   ./scripts/make_py2_venv.sh
-
-If you change the Pipfile or the package requirements, please make sure
-to regenerate the lockfiles for both Python versions. The easiest way to
-do so is with the following script:
-
-::
-
-   ./scripts/make_pipenv_lockfiles.sh
-
-Then, re-run
-
-::
-
-   pipenv sync --dev
-
-to install the relevant dependencies.
-
```

Comment:
Remember to update the readthedocs rst files too. Maybe we should just update readthedocs at this point. We are only missing the query printing and the execution model sections.

Reply:
I think I updated both `CONTRIBUTING.rst` files. Is there another file I should update?

Reply:
Sorry, I missed the changes in the bottom.

Reply:
Sorry, I missed the changes in the bottom.